src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        Colors figureColor = Colors.values()[index];
+        return figureColor.toString();
```

Comment:
Instead of using `toString()`, you should use the `name()` method to get the String representation of the enum constants. This ensures that the correct name is returned even if `toString()` is overridden in the enum.

Reply:
Instead of using toString(), you should use the name() method to get the String representation of the enum constants. This ensures that the correct name is returned even if toString() is overridden in the enum.